File: storage/data_transpiled/at_coder/arc021/A/4819526.py
def main ( ) :
    import sys
    from time import sleep
    sc = Scanner ( )
    a = [ ]
    for i in range ( 4 ) :
        for j in range ( 4 ) :
            a.append ( sc.randint ( 0 , 4 ) )
            if i != 0 :
                if a [ i ] [ j ] == a [ i - 1 ] [ j ] :
                    print ( "CONTINUE" )
                    return
            if j != 0 :
                if a [ i ] [ j ] == a [ i ] [ j - 1 ] :
                    print ( "CONTINUE" )
                    return
    print ( "GAMEOVER" )

File: storage/data_original/at_coder/arc021/A/4819526.java
import java.util.*;
public class Main{
	public static void main(String[]args) {
		Scanner sc = new Scanner(System.in);
		int[][]a=new int[4][4];
		for(int i=0;i<4;i++) {
			for(int j=0;j<4;j++) {
				a[i][j]=sc.nextInt();
				if(i!=0) {
					if(a[i][j]==a[i-1][j]) {
						System.out.println("CONTINUE");
						return;
					}
				}
				if(j!=0) {
					if(a[i][j]==a[i][j-1]) {
						System.out.println("CONTINUE");
						return;
					}
				}	
			}
		}
		System.out.println("GAMEOVER");
	}
}